src/data/html-quiz.ts

```diff
@@ -1,4 +1,16 @@
 const htmlQuiz = [
+  {
+    Question:
+    "Which tag is used to insert favicon?",
+    Answer: "<link>",
+    Distractor1: "<fav>",
+    Distractor2: "<favicon>",
+    Distractor3: "<icon>",
+    Explanation:
+      "In HTML, <link> tag is used inside the <head> tag",
+    Link:
+      "https://www.w3schools.com/html/html_favicon.asp",
```

Comment:
Please only use freeCodeCamp resources or official documentation.

```suggestion
      "https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link",
```